README.md

```diff
@@ -1,10 +1,8 @@
 # Murray Clubhouse Desk Booking App
 
-[TOC]
```

Comment:
Minor - please generate a TOC 
```
- [[#Useful links|Useful links]]
- [[#Cloning the project|Cloning the project]]
- [[#Setting Up Env Files|Setting Up Env Files]]
- [[#Getting the app running|Getting the app running]]
	- [[#Getting the app running#iOS Profiles|iOS Profiles]]
- [[#Deploying to firebase distribution|Deploying to firebase distribution]]
- [[#Build variant/Configurations/Schemes/Flavours|Build variant/Configurations/Schemes/Flavours]]
	- [[#Build variant/Configurations/Schemes/Flavours#iOS|iOS]]
	- [[#Build variant/Configurations/Schemes/Flavours#Android|Android]]
- [[#Setting up Firebase emulators|Setting up Firebase emulators]]
- [[#Running the Firebase Emulator for Firebase Functions|Running the Firebase Emulator for Firebase Functions]]
	- [[#Running the Firebase Emulator for Firebase Functions#Firebase Admin SDK|Firebase Admin SDK]]
- [[#Set up Firebase Functions locally|Set up Firebase Functions locally]]
- [[#Running the Firebase function unit tests|Running the Firebase function unit tests]]
- [[#Running the unit tests|Running the unit tests]]
- [[#I have an M1/M2 macbook (rather than intel) and it's not working|I have an M1/M2 macbook (rather than intel) and it's not working]]
- [[#I'm seeing error messages|I'm seeing error messages]]
	- [[#I'm seeing error messages#If you're a fish shell user|If you're a fish shell user]]
- [[#Unknown ruby interpreter version (do not know how to handle): >=XXXX|Unknown ruby interpreter version (do not know how to handle): >=XXXX]]
- [[#"Help! I've tried everything and I'm all out of ideas."|"Help! I've tried everything and I'm all out of ideas."]]
- [[#`Watchman crawl failed`|`Watchman crawl failed`]]
```